lib/promscrape/discoveryutils/client.go

```diff
@@ -33,10 +33,11 @@ func GetHTTPClient() *http.Client {
 
 // Client is http client, which talks to the given apiServer.
 type Client struct {
-	hc        *fasthttp.HostClient
-	ac        *promauth.Config
-	apiServer string
-	hostPort  string
+	hc          *fasthttp.HostClient
+	watchClient *fasthttp.HostClient
```

Comment:
could you rename watchClient to `blockingClient` and add the following comment to it:
```
blockingClient is used for performing long-polling requests.
```

Reply:
done